### ISSUE TITLE
Blank screen when creating an email from a theme

### ISSUE THREAD

**Issue by nunesf** [mautic/mautic#5859]
Bug report

## Description:
I created 2 e-mail themes in the previous version of Mautic. Now I created a new theme, I made the installation inside of the mautic itself, but when I create a new email, selecting one of my themes, when I open the constructor, I see only a blank screen.

I did several tests with my themes and nothing had any effect, when I selected the standard themes also did not work, I have a general bug when using themes in the emails.

What I noticed is that whenever I select a theme to create an email, a 500 error occurs in that url:
https://mydomain.com/s/emails/builder/new_0fc99af7529cfea36dc0623c619cba2f0e3d5d6a?template=cards&_=1521918037990&mauticUserLastActive=1&mauticLastNotificationId=

Request URL: https://mydomain.com/s/emails/builder/new_0fc99af7529cfea36dc0623c619cba2f0e3d5d6a?template=cards&_=1521918037990&mauticUserLastActive=1&mauticLastNotificationId=
Request Method: GET
Status Code: 500 Internal Server Error
Remote Address: 52.201.117.186:443
Referrer Policy: no-referrer-when-downgrade

GET /s/emails/builder/new_0fc99af7529cfea36dc0623c619cba2f0e3d5d6a?template=cards&_=1521918037990&mauticUserLastActive=1&mauticLastNotificationId= HTTP/1.1
Host: mydomain.com
Connection: keep-alive
Accept: */*
X-Requested-With: XMLHttpRequest
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.181 Safari/537.36
Referer: https://mydomain.com/s/emails/new
Accept-Encoding: gzip, deflate, br
Accept-Language: en-US,en;q=0.9,pt;q=0.8
Cookie: _ga=GA1.2.983991547.1515755611; __zlcmid=kRh2ZVh9ETW4e7; messagesUtk=1113abcac2c84676bdb7a160b5449e79; hubspotutk=6367072c213b25a5f06466dbb8bd2ded; __hstc=41907133.6367072c213b25a5f06466dbb8bd2ded.1515958079019.1520543685951.1521480461402.12; 9ded1821a5fd0bc510ba87fcabaf33d1=pefaru1cpif511ui4bi2l5opc3

{"error":{"code":500,"text":"Internal Server Error","exception":"","trace":""},"route":"\/s\/emails\/builder\/new_0fc99af7529cfea36dc0623c619cba2f0e3d5d6a","flashes":"","notifications":{"content":"\n","lastId":"","hasNewNotifications":false,"updateAvailable":true},"browserNotifications":[],"newContent":"\u003Cdiv class=\u0022pa-20 mautibot-error\u0022\u003E\n    \u003Cdiv class=\u0022row mt-lg pa-md\u0022\u003E\n        \n        \u003Cdiv class=\u0022mautibot-image col-xs-4 col-md-3\u0022\u003E\n            \u003Cimg class=\u0022img-responsive\u0022 src=\u0022data:image\/png;base64,iVBORw0KGgoAAAANSUhEUgAAAMgAAADZCAYAAABl0n+gAAAABGdBTUEAALGPC\/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAACXBIWXMAAAsTAAALEwEAmpwYAAABWWlUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iWE1QIENvcmUgNS40LjAiPgogICA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyI+CiAgICAgICAgIDx0aWZmOk9yaWVudGF0aW9uPjE8L3RpZmY6T3JpZW50YXRpb24+CiAgICAgIDwvcmRmOkRlc2NyaXB0aW9uPgogICA8L3JkZjpSREY+CjwveDp4bXBtZXRhPgpMwidZAABAAElEQVR4Aey9CZhlR3UmGO+93DMrl9r3ylq0lpAEAgkQyAkYkNnxgvfpD+xu2932eNx2j+3pb+x2f223F5r2Mqa9sUxj+7PHNtDGBmRsKHYBEkgCoVKpSlWqTao9Myv3fMv8\/38i4sZ972VWZlVmViEUme\/eiLPFiRNxbiw37r0F9+0VCihu8ftw+BvnKk2KXrxxy5aBckvnjmKpurFWrW6v1grbwLSjUHT9tVphoOBqq2vODdRqtXbwtzlXw6\/Q4mVVgZspFAozhRrPbgzpC8Dj50ZdrXoBBEcg61SxWDxeqNXOuNnZE619fWceffTRmSb6EFQcwm8fMnLOVfFjYPy5sAIWYIN5tofikDUwOkTasAo7d+7c3lIt3OIK1b01V7wN6D01Vxss1AqrXcG1oKHnbAOnUDqcc8gmicBfgDD+M\/AkKZBVrdWo00XQPYPzU8j7MM6PwXmeLNZqT0zXak8dOXJkCrD6UBqCqH3mMBRnitVTPZe+Ygv4artiOdeagMKQc6V91kvExnP91q1bqsXWl9aKtbvRaO+C0nvxW1WEI6iVRQcAVI2uUAUdUMCrlat9pzaTC1kGsemT1UA68aAk2r\/BkYJPoB8xf2FCP2bKQDL8JgE8Bsw34UffKLrSQ5WKe+TJ408eBklZhNkhdZjQy2TY52KXbYG0si9byDXEGBpdHD5dPzh4I1rMvWiLr0Wru7NQLK6mvtYIGUN\/4YoVV4QT1GpFDINgkwpOswDMADYLmmkwYARUs7an5k5WH3JG5GirgNFXoQ3tm3F\/diXEg3qUgB+VsFGTEgARAboCiOkzdY5TrY4DerDmCl8D3\/0Yon15Ynb28ZMnT06ILztwyEdZVNhkZrjnYouwQK5uF8F3rZGyQYUG4QYHBze2OvdmNKbvRRO8Gw2tkwpXOUPAeKqACYY5Q7lQq07iNwLkFHsJR\/8ptPS7Yksvfj34dbpiqQM\/NPQipxtFtlz84EsUyoCRkhyuSseCU1WmXbUy5apl\/CrjOI8BdhG\/c6CrgBc8hR6I6savA3G251AVaNP0m+xQgyPAJ+Rd6Owyp6lVa2XofAhudz\/4P40M7n\/iqaf2kxm\/EGgbhuecxeywqGOolUUxXUPEOcfYs2PHS9DE34629hY0pHXUk06BHgINCS2rVimioRaq1bOEuWLrNtfSvsm1dK7Fud+V2la5UmuXK5Xa0XBb7MfWzJ8acGYuxtJWyLwUrHEDByx6nJqcpgxHmcFv2lVmJ1x55qKrzIy6yvQFxM+46uxp0I0hmxLyHMCvC8WAi0NHtWv1XMqNB46\/WCqWqMQDfwzVKry9UHgYJJ+q1Ur\/3DI+8pXHz569KKQdnnOWxBgLiWY1vhDqa4eGLYdBY57dg4P3oiA\/i6ZzL4ZQaGxomBw28aJbKxer5fPoJUbR+Le61u6drq17s2ulU7T1ulJLB8jQbnSBpkg2bM0BLM50CEk0gJqeZdVgWjZgUuGgCBM+DzhPhb2Nd5pZOszkGfyegSMdR+km4SOrXKHUB\/W4aBZ4Y2dAjSjMHAbzruAw7HRAfwgjyH3oMz9WmJ369IGTJ88CGMJzzhIsMc9ZVTcP\/lpDFYZs8q1J6q7tu15VKNR+GdOHV1FRjpuw+FTBmKRUK48WapUzrtg26NpWXe86Vm13bV1r4CQc1vghuhzBNza2J4ZokRgx+BUdvfCQR8yHzkNfR15yHjgmnKZannbl2TE3O3XBzU6cwu+kK089BdwonAXDv1Iv6DHcY1DvouuEUoSwh4E8OgsD4bTNM1hu+Bd0Sn\/fMjPzL\/tPnDgnhB3oLNQuCkpw39bRpWwFy21IVqIm33u2b7+5Vij+OiqfQykOLdAgSmwYpWqZK6Y1OMULXGf\/da69G0OoNgxZOO\/lkIcNSldXkKn0V9sEdc6D8qhRB8eBvhyalacvupnJc25m\/

Mautic version 2.12.2
PHP Version 5.6.30

### Steps to reproduce:
1. 
2. 
 
### Log errors: 
[2018-03-24 15:39:42] mautic.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The directory "/opt/bitnami/apps/mautic/htdocs/app/cache/prod/annotations" is not writable." at /opt/bitnami/apps/mautic/htdocs/vendor/doctrine/cache/lib/Doctrine/Common/Cache/FileCache.php line 92 {"exception":"[object] (InvalidArgumentException(code: 0): The directory \"/opt/bitnami/apps/mautic/htdocs/app/cache/prod/annotations\" is not writable. at /opt/bitnami/apps/mautic/htdocs/vendor/doctrine/cache/lib/Doctrine/Common/Cache/FileCache.php:92)"} []
[2018-03-24 15:44:42] mautic.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The directory "/opt/bitnami/apps/mautic/htdocs/app/cache/prod/annotations" is not writable." at /opt/bitnami/apps/mautic/htdocs/vendor/doctrine/cache/lib/Doctrine/Common/Cache/FileCache.php line 92 {"exception":"[object] (InvalidArgumentException(code: 0): The directory \"/opt/bitnami/apps/mautic/htdocs/app/cache/prod/annotations\" is not writable. at /opt/bitnami/apps/mautic/htdocs/vendor/doctrine/cache/lib/Doctrine/Common/Cache/FileCache.php:92)"} []
[2018-03-24 15:49:43] mautic.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The directory "/opt/bitnami/apps/mautic/htdocs/app/cache/prod/annotations" is not writable." at /opt/bitnami/apps/mautic/htdocs/vendor/doctrine/cache/lib/Doctrine/Common/Cache/FileCache.php line 92 {"exception":"[object] (InvalidArgumentException(code: 0): The directory \"/opt/bitnami/apps/mautic/htdocs/app/cache/prod/annotations\" is not writable. at /opt/bitnami/apps/mautic/htdocs/vendor/doctrine/cache/lib/Doctrine/Common/Cache/FileCache.php:92)"} []
[2018-03-24 15:53:47] mautic.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the cache directory (/opt/bitnami/apps/mautic/htdocs/app/cache/prod/twig/72)." at /opt/bitnami/apps/mautic/htdocs/vendor/twig/twig/lib/Twig/Cache/Filesystem.php line 57 {"exception":"[object] (RuntimeException(code: 0): Unable to create the cache directory (/opt/bitnami/apps/mautic/htdocs/app/cache/prod/twig/72). at /opt/bitnami/apps/mautic/htdocs/vendor/twig/twig/lib/Twig/Cache/Filesystem.php:57)"} []
[2018-03-24 15:53:52] mautic.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the cache directory (/opt/bitnami/apps/mautic/htdocs/app/cache/prod/twig/01)." at /opt/bitnami/apps/mautic/htdocs/vendor/twig/twig/lib/Twig/Cache/Filesystem.php line 57 {"exception":"[object] (RuntimeException(code: 0): Unable to create the cache directory (/opt/bitnami/apps/mautic/htdocs/app/cache/prod/twig/01). at /opt/bitnami/apps/mautic/htdocs/vendor/twig/twig/lib/Twig/Cache/Filesystem.php:57)"} []

**Comment by gustavofoa** [comment]
I am with the same error in my mautic instalation.
Is there any update about this issue?

**Comment by diegofcv** [comment]
I had the same situation here. The problem was that I made a clean up in the cache (php app/console cache:clear) as the root user on the machine. Once I did that, the directory /app/cache/prod was re-created with wrong ownership (root), making it impossible to Mautic to write the needed cache files in that place, turning the white screen problem to happen. 

Fixing the permissions (with "chown -R user.group prod/") did fix the problem immediately. 

By the errors above, posted by @nunesf , it should be the same situation, therefore probably not a bug.
